documentation/reference/CMakeLists.txt

```diff
@@ -9,10 +9,22 @@ IF (BUILD_DOCUMENTATION)
     #
     # Configure the script and the doxyfile, then add target
     #
+    # XXXX: This should not be done in this manner, or at least it should break when dot is not found.
+    #       The way to do that is:
+    #       find_package(Doxygen
+    #                    REQUIRED dot
+    #                    OPTIONAL_COMPONENTS gnuplot)
+    #       if(NOT DOXYGEN_DOT_FOUND)
+    #            message(WARNING "Graphviz doesn't seem to be installed.\
+    #                             Doxygen will not be able to generate graphs.\
+    #                             Consider installing this package.")
+    #       endif(NOT DOXYGEN_DOT_FOUND)
+    #
```

Comment:
shall we clean this?

Reply:
No. This is here to point how this should be done. Either we change it and do it properly or add XXX comment, move on and let someone else pick it up (or Sunday afternoon).

Reply:
I mean

Reply:
+ it should be done at least one layer above. (in your other [comment](https://github.com/openmeeg/openmeeg/pull/289/files/a5ccaffde385aecc7c909b170c9e2fba4933c54a#r178302874) documentation/CMakeLists.txt)

Reply:
But if we end up moving to fully documenting with Sphinx without doxygen, there would be no need for fixing it.